react/react_testing/introduction_to_react_testing.md

```diff
@@ -164,6 +164,7 @@ The other issue with snapshots is false negatives. Even the most insignificant o
 
 1. Take a glance at all of the available query methods on [the React Testing Library's cheatsheet page](https://testing-library.com/docs/dom-testing-library/cheatsheet/). There's no need to use them all, but it's optimal to employ a specific method for a specific query. If none of the query methods suffice, there's an option to use test ids. Learn about test ids on [the React Testing Library's test id docs](https://testing-library.com/docs/queries/bytestid/).
 1. Read [the userEvent API docs](https://testing-library.com/docs/user-event/intro) to get a feel of how to achieve user simulation.
+1. [Testing Implementation Details](https://kentcdodds.com/blog/testing-implementation-details) article by Kent C. Dodds prefaces with an Enzyme example which perfectly illustrates the problems React Testing Library strives to solve.
```

Comment:
Regarding the order, I don't have particularly strong feelings about where it goes, to be honest. Third made most sense to me at a glance, but first is fine too so if you feel it makes a lot more sense first, let's go with that :)

It would be best at least to have this worded so that we're more summarising the purpose of the article, which is about the dangers of testing implementation. The Enzyme thing should just be a side note so people are expecting it (so people don't go "Are we meant to use Enzyme?"/"I'm confused, what is Enzyme?" before they actually read the article properly).

e.g. something like
```suggestion
1. [Testing Implementation Details](https://kentcdodds.com/blog/testing-implementation-details) by Kent C. Dodds shows us how we can reduce false test results and inflexible tests by avoiding testing the implementation of things. Note that while it starts with examples using the Enzyme testing library, the concepts are the primary focus.
```

Reply:
I find that understanding the 'why' first is crucial. I leave the exact wording to you since my English isn't ideal for such purposes.

Reply:
> I find that understanding the 'why' first is crucial. I leave the exact wording to you since my English isn't ideal for such purposes.

Sure thing. In that case, let's use the suggested wording above, and make it the first assignment item :muscle: It's a brilliant article, great suggestion.